package.json

```diff
@@ -35,6 +35,11 @@
           "type": "number",
           "default": 0,
           "description": "How many minutes before a scheduled notification should the notification fire?"
+        },
+        "kaleidofoam.nativeNotifications": {
+          "type": "boolean",
+          "default": false,
+          "description": "Use native vscode (true) or backwards compatible node-notifier package (false) notification system"
```

Comment:
Thanks so much for doing this! I think this is a great addition. One suggestion though: what if instead of doing `nativeNotifications` we had a `notifications` setting with an enum? I figure it opens up the space for additional types in the future and makes it more obvious what a value should do. Something like this:

```json
"kaleidofoam.notifications": {
  "type": "string",
  "default": "system",
  "enum": ["system", "vscode", "none"],
  "enumDescriptions": [
    "Attempt to use system notifications (uses node-notifier)",
    "Use VS Code notifications",
    "Disable notifications"
  ]
}
```

Reply:
Absolutely, good idea. I'll update the PR 👍

Are you happy prioritising backwards compatibility i.e. using system as default?